driver-legacy/src/main/com/mongodb/Mongo.java

```diff
@@ -751,8 +747,8 @@ private static ClusterSettings getClusterSettings(final List<ServerAddress> seed
                 .hosts(new ArrayList<ServerAddress>(seedList))
                 .mode(clusterConnectionMode)
                 .requiredReplicaSetName(options.getRequiredReplicaSetName())
-                .serverSelectionTimeout(options.getServerSelectionTimeout(), MILLISECONDS)
```

Comment:
Why remove the serverSelectionTimeout setting?

Reply:
Inadvertent change.